guides/common/modules/proc_configuring-server-to-synchronize-content-over-a-network.adoc

```diff
@@ -8,12 +8,20 @@ Configure a downstream {ProjectServer} to synchronize repositories from a connec
 * You imported the subscriptions manifest on both the upstream and downstream {ProjectServer}.
 * You enabled the required repositories for the organization on the upstream {ProjectServer}.
 * You have user permissions to create product repositories and organizations on the downstream {ProjectServer}.
+* You have imported the SSL certificate of the upstream {ProjectServer} on the downstream {ProjectServer} using the contents of `http://_upstream-{foreman-example-com}_/pub/katello-server-ca.crt`.
+For more information, see {ContentManagementDocURL}Importing_Custom_SSL_Certificates_content-management[Importing SSL Certificates] in the _Content Management Guide_.
+* The upstream user is an admin or has the following permissions:
+** View Organization
+** Edit Organization (to download the debug certificate)
+** View Content Views
+** View Lifecycle Environments
+** View Products
```

Comment:
@rmynar, can you confirm all these credentials are necessary? I was testing non-admin only with
* View Organization
* Edit Organization (to download the debug certificate)
* View Products

Reply:
The permissions "view_content_views" and "view_lifecycle_environments" are necessary if users don't use default values for Content view label ("Default Organization View") or Lifecyctle Environment Label ("Library").
Note that permission names are written in lowercase and joined by underscores (view_organizations, edit_organizations, view_products, view_lifecycle_environments and view_content_views)
It would be nice to don't list all the permissions but use just role name - I entered a RFE: https://bugzilla.redhat.com/show_bug.cgi?id=2087748